packages/office-ui-fabric-react/src/components/Slider/Slider.types.ts

```diff
@@ -12,6 +12,10 @@ export interface ISlider {
   focus: () => void;
 }
 
+export interface ISliderMarksArrayFormat {
+  value: number;
+  label: string;
```

Comment:
It would very nice to have the interface itself documented with a clear description, added a `{@docCategory Slider}` as with other APIs referring to Slider control and also each property needs to be documented.

Reply:
ok just added it feel free to check if this is what you had in mind

Reply:
Wrong placement. All documentation needs to be above the thing it documents

Reply:
ok just added it to the top

Reply:
@sneharoy321 you added it at the top but you shouldn't have moved everything. So they way docs work each interface and each property on the interface needs to be documented. You don't throw everything above the interface name. Can you sync with @aneeshack4 on this please.